packages/core/src/managed/config/DeliveryConfig.tsx

```diff
@@ -1,24 +1,65 @@
 import React from 'react';
+
 import { RelativeTimestamp } from '../RelativeTimestamp';
+import {
+  FetchApplicationManagementDataDocument,
+  FetchApplicationManagementDataQueryVariables,
+  useImportDeliveryConfigMutation,
+} from '../graphql/graphql-sdk';
+import { useApplicationContextSafe } from '../../presentation/hooks/useApplicationContext.hook';
 import { YamlViewer } from '../utils/YamlViewer';
+import { NotifierService, Spinner } from '../../widgets';
 
 interface IDeliveryConfigProps {
   config?: string;
   updatedAt?: string;
   isProcessed?: boolean;
 }
 
+const ReImportConfig = () => {
+  const appName = useApplicationContextSafe().name;
+  const refetchVariables: FetchApplicationManagementDataQueryVariables = { appName };
+  const [importDeliveryConfig, { error, loading }] = useImportDeliveryConfigMutation({
+    variables: { application: appName },
+    refetchQueries: [{ query: FetchApplicationManagementDataDocument, variables: refetchVariables }],
+  });
+
+  React.useEffect(() => {
+    if (!error) return;
+    NotifierService.publish({
```

Comment:
I'm assuming this has an effect on the UI? Can you please post a sample of what that looks like?

Reply:
This is the button right notifications you see when an error happens

Reply:
![image](https://user-images.githubusercontent.com/17147717/131861465-d4bd613c-b58b-4511-ac95-49275691df65.png)